src/api/students.js

```diff
@@ -20,7 +20,7 @@ export default ({ config, db }) => resource({
 	
```

Comment:
add the `getStudents` function that returns an array of all the students in the school